### PR TITLE
Remove the bugzilla-annotations feature

### PR DESCRIPTION
**Pull request by sideshowbarker**
Relates to https://github.com/whatwg/html-build/issues/177

### CODE REVIEW

**Comment by sideshowbarker** [comment]
Thanks — I went back and removed the rest and amended the commit. So the code on the branch is ready for re-review

File: src/wattsi.pas [hunk 1]
```diff
@@ -2793,10 +2603,6 @@ function Main(): Boolean;
                for BrowserIndex := Low(Features[ID].FirstGoodVersion) to High(Features[ID].FirstGoodVersion) do
                   if (Features[ID].FirstGoodVersion[BrowserIndex] <> '') then
                      Write(' ', Browsers[BrowserIndex].Name, ' ', Features[ID].FirstGoodVersion[BrowserIndex], '+ ');
-               Writeln();
```
Comment (annevk) [inline src/wattsi.pas]: Should this be removed? Seems you might not have a newline without it.

Comment (sideshowbarker) [reply src/wattsi.pas]: Yep you’re right — thanks for catching that. Fixed and pushed to the branch

File: src/wattsi.pas [hunk 2]
```diff
@@ -1319,15 +1298,6 @@    TCrossReferences = record
          end
          else
          begin
-            if (Variant <> vDEV) then
```
Comment (domenic) [inline src/wattsi.pas]: Wait, we still need this for caniuse warnings! Those are actionable :)

Comment (sideshowbarker) [reply src/wattsi.pas]: >```
>-           if (Variant <> vDEV) then
>```
>
> Wait, we still need this for caniuse warnings! Those are actionable :)

We still get the caniuse warnings, regardless. That conditional block was just creating a separate code path that handled reporting of both the bugzilla annotations and caniuse warnings together. The `else` for that conditional block repeats the exact same handling for just caniuse case. Here’s the full context:

```
-            if (Variant <> vDEV) then
-            begin
-               Warn('Could not find ID ' + ID + ' for annotation that uses URLs:');
-               for Bug in Feature.Bugs do
-                  Writeln('   ', Bug.URL);
-               if (Feature.CanIUseCode <> '') then
-                  Writeln('   https://caniuse.com/#feat=', Feature.CanIUseCode);
-            end
-            else
             if (Feature.CanIUseCode <> '') then
             begin
                Warn('Could not find ID ' + ID + ' for annotation that uses URLs:');
                Writeln('   https://caniuse.com/#feat=', Feature.CanIUseCode);
             end;
```

**Pull request merged**